Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionMicrosoftWindowsEventFirewall.yaml

```diff
@@ -142,6 +148,7 @@ ParserQuery: |
       let SecurityEvent_5154_5155_5158_5159 =
       SecurityEvent | where not(disabled)
           | where EventID in (5154, 5155, 5158, 5159)
+          | extend EventResult = iff(EventID in (5154, 5158), "Success", "Failure")
```

Comment:
Since it is needed only for the last filter, I prefer to put it at the end of the prefiltering. Sort of phase II prefilter.